port/unix/omrsysinfo.c

```diff
@@ -1718,7 +1727,12 @@ omrsysinfo_get_limit(struct OMRPortLibrary *portLibrary, uint32_t resourceID, ui
 	Trc_PRT_sysinfo_get_limit_Entered(resourceID);
 
 	if (OMRPORT_RESOURCE_ADDRESS_SPACE == resourceRequested) {
+#if !defined(OMRZTPF)
 		resource = RLIMIT_AS;
+#else /* !defined(OMRZTPF) */
+		resource = OMRPORT_LIMIT_UNKNOWN;
+		return resource;
```

Comment:
Why the early return in this case but not in the set limit case?  Is the rest of this code applicable to ztpf as it would maintain the single point of return,...